option_parser/src/lib.rs

```diff
@@ -170,3 +170,26 @@ impl FromStr for ByteSized {
         }))
     }
 }
+
+pub struct VectoredByteSized(pub Vec<u64>);
+
+pub enum VectoredByteSizedParseError {
+    ParseItem(ByteSizedParseError),
+}
+
+impl FromStr for VectoredByteSized {
+    type Err = VectoredByteSizedParseError;
+
+    fn from_str(s: &str) -> std::result::Result<Self, Self::Err> {
+        let mut vec_byte_sized = Vec::new();
+        let items_list: Vec<&str> = s.trim().split(':').collect();
```

Comment:
That may be just me, but when I read `--host-numa-nodes=1:4`, I feel the user is asking for the [1:4] range. It may be just me...Wouldn't `;` be less confusing?

Reply:
Yes fine with me, I'll make the change.

Reply:
Oh no we can't! `bash` cuts the command line because of `;`.